documentation/concepts/check.md

```diff
@@ -4,25 +4,88 @@ menu: check
 subtitle: Check
 ---
 # Check
-{% include enterprise.html %}
+<strong>The `check` command is currently in beta, and only available in the Enterprise edition of Flyway. </strong>
+<br>{% include enterprise.html %}
```

Comment:
In general we should aim to use markdown syntax in `md` files as much as possible

The `div` below doesn't have a markdown alternative, but `strong` can be replaced by wrapping the line in `**` to make the text **bold**